orange/src/test/java/com/hartwig/hmftools/orange/conversion/OrangeConversionTest.java

```diff
@@ -0,0 +1,46 @@
+package com.hartwig.hmftools.orange.conversion;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+
+import com.hartwig.hmftools.datamodel.virus.AnnotatedVirus;
+import com.hartwig.hmftools.datamodel.virus.VirusInterpreterData;
+
+import org.junit.Test;
+
+public class OrangeConversionTest {
+
+    @Test
+    public void shouldConvertAnnotatedVirus() {
+        assertTrue(OrangeConversion.convert(TestCommonDatamodelFactory.emptyVirusInterpreterData()).allViruses().isEmpty());
+        assertEqualsValue(OrangeConversion.convert(TestCommonDatamodelFactory.minimalVirusInterpreterData()),
+                TestCommonDatamodelFactory.minimalVirusInterpreterData());
+        assertEqualsValue(OrangeConversion.convert(TestCommonDatamodelFactory.exhaustiveVirusInterpreterData()),
+                TestCommonDatamodelFactory.exhaustiveVirusInterpreterData());
+    }
+
+    private static void assertEqualsValue(VirusInterpreterData converted, com.hartwig.hmftools.common.virus.VirusInterpreterData input) {
```

Comment:
I think it's more conventional to provide the expected value as the first parameter. The effect is mostly the same, but it can impact failure messages.